proposals/csharp-10.0/using-alias-improvements.md

```diff
@@ -0,0 +1,117 @@
+﻿# Using alias improvements
+
+## Summary
+
+Extend `using` aliases to allow generic type aliases and additional alias target types.
+
+## Generic type aliases
+
+Type aliases may include type parameters.
+```C#
+using MyList<T> = System.Collections.Generic.List<T>; // ok
+```
+
+Type parameters do not need to be referenced in the target.
+```C#
+using MyType1<T> = System.String; // ok
+using MyType2<T, U> = T;          // ok
+```
+
+Type parameter references in an alias target have the same restrictions as type parameter references in other generic contexts.
+```C#
+using MyType3<T, U> = T.U;  // error: type or namespace 'T' not found
+using MyType3<T, U> = T<U>; // error: type or namespace 'T<>' not found
+```
+
+The same name may be used for multiple aliases with distinct arity.
+```C#
+using MyDictionary = System.Collections.Generic.Dictionary<string, object>;
+using MyDictionary<T> = System.Collections.Generic.Dictionary<string, T>; // ok
+using MyDictionary<U> = System.Collections.Generic.Dictionary<string, U>; // error: 'MyDictionary<>' already defined
+```
+
+A partially bound type is not valid.
+The `using` aliases below are both valid, and the use of `MyList<>` is valid because the expansion `List<>` is an unbound type, but the use of `MyDictionary<>` invalid because `Dictionary<string,>` is a partially bound type.
```

Comment:
Consider adding an example of a valid use of `MyDictionary`. This section is still starting with the words "A partially bound type is not valid.", which doesn't really address the confusion.